lab-yunjoo/app/component/list/list.html

```diff
@@ -0,0 +1,10 @@
+<div class="app-list">
+  <i class="fa fa-trash-o fa-2x" ng-click="listCtrl.deleteList()"></i>
+  <h2>{{listCtrl.list.name}}</h2>
+  <ul>
+    <li>
+      <app-note-li ng-repeat="item in listCtrl.list.notes" note="item" delete-note="listCtrl.deleteNote(noteId)"></app-note-li>
```

Comment:
Your ng-repeat should be on the li element. As is, you are nesting the li's you are repeating inside of a single li inside of a ul, which doesn't make any sense.